core/src/main/java/org/apache/carbondata/core/datastore/chunk/AbstractRawColumnChunk.java

```diff
@@ -54,6 +55,15 @@ public AbstractRawColumnChunk(int columnIndex, ByteBuffer rawData, long offSet,
     this.length = length;
   }
 
+  public ColumnPagesBloomFilter getPageBloomFilter() {
+    // page bloom is disabled for current column in this blocklet
+    if (!dataChunkV3.isSetPage_bloom_chunk()) {
+      return null;
+    } else {
```

Comment:
no need of else statement directly add return new ColumnPagesBloomFilter(dataChunkV3.page_bloom_chunk);

Reply:
if page bloom is disabled for current column, the page_bloom_chunk will be NULL, no need to new a useless ColumnPagesBloomFilter object, and we still need to decide whether the object is efftive for pruning

Reply:
```suggestion
    if (!dataChunkV3.isSetPage_bloom_chunk()) {
      return null;
    } 
    return new ColumnPagesBloomFilter(dataChunkV3.page_bloom_chunk);
```

He means like above

Reply:
```suggestion
 if (!dataChunkV3.isSetPage_bloom_chunk()) {
      return null;
    } 
    return new ColumnPagesBloomFilter(dataChunkV3.page_bloom_chunk);
```
he means like above

Reply:
OK, thanks @ajantha-bhat